pkg/scheduler/plugins/nodeorder/nodeorder.go

```diff
@@ -255,23 +255,23 @@ func (pp *nodeOrderPlugin) OnSessionOpen(ssn *framework.Session) {
 			return 0, err
 		}
 		// If leastReqWeight in provided, host.Score is multiplied with weight, if not, host.Score is added to total score.
-		score = score + (host.Score * weight.leastReqWeight)
+		score = score + float64(host.Score*weight.leastReqWeight)
```

Comment:
It gets `int` at each step, where does it get `float` value?

Reply:
We don't have a function with float value now.

But I think the calculation of leastReq should return float and others may meet the same situation. So I just provide a way to choose the best fist node but not a set of similar node with the same score.